### DISCUSSION TITLE
Foliate won't start up in Manjaro KDE Plasma

### DISCUSSION THREAD

**Discussion by WisdomMoon** [johnfactotum/foliate#956] [Q&A]
I've been using foliate for quite a while, successfully on Manjaro KDE plasma, but something has changed, because in the past few days,  I cannot launch it.  Foliate will launch once when I first boot or reboot my system, but every subsequent time I try to launch, it doesn't work.

I've tried various things, such as reinstalling from Manjaro repositories, and I've even compiled it from source and installed it, and it still won't start. I suspect it might be a problem with GTK 3 somewhere along the line. Are there any log files I can look at to see what the problem might be, or do you have any advice about how to diagnose what the problem is? 

Thanks for your help!

**Comment by johnfactotum** [comment]
Try running it from a terminal and look at the output. There's also the `--verbose` option which will print more debug info.

**Reply by WisdomMoon** [reply]
Thanks for your reply, when I run it from a terminal, even with the --verbose option, there's nothing. The cursor just sits there, blinking.

I'm using the command  ./com.github.johnfactotum.Foliate --verbose in the /usr/bin directory.

**Reply by johnfactotum** [reply]
> I've been using foliate for quite a while, successfully on Manjaro KDE plasma, but something has changed, because in the past few days, I cannot launch it. 

If you've upgraded your system recently, maybe you can try downgrading.

**Reply by WisdomMoon** [reply]
Manjaro is a rolling release and there are lots of small updates happening every day, so it's not possible to downgrade because there's no way to know what update broke the app and when. Do you have any ideas about what kinds of things might stop foliate from launching? As I wrote, it does launch if I reboot my system, but any subsequent attempt to launch it fails. 

Thank you for creating foliate, It's great and I have used it a lot.  I think it's the best e-reader on Linux.

**Comment by WisdomMoon** [comment]
My current workaround is using foliate in a virtual machine, but if anyone has any ideas regarding my questions above, please get in touch. Thanks!

**Comment by WisdomMoon** [comment]
UPDATE: I saw a YouTube video of an Arch user complaining that his rolling updates broke vital apps all the time, such as Gimp and Kdenlive, and so he installed app images of those apps which just worked. I installed the Snap version of Foliate, and now it works great. This is a tip for anyone who  might be having problems similar to my own.
